app/script/util/DebugUtil.js

```diff
@@ -58,6 +58,13 @@ z.util.DebugUtil = class DebugUtil {
       .then(() => this.logger.log(`Corrupted Session ID '${sessionId}'`));
   }
 
+  getLastMessagesFromDatabase(amount = 10, conversationId = wire.app.repository.conversation.active_conversation().id) {
+    return wire.app.repository.storage.storageService.db.events.toArray(records => {
```

Comment:
Would be better to inject the `storageService` in the constructor of the Util instead on relying on global variable